modules/drivers/microphone/microphone_component.cc

```diff
@@ -72,30 +71,29 @@ bool MicrophoneComponent::Init() {
   return true;
 }
 
+void MicrophoneComponent::fill_channel_data(int chunk_i) {
+  // next index of channel data to be filled
+  int pos = chunk_i * chunk_ * sample_width_;
+  for (int buff_i = 0; buff_i < chunk_size_; pos += sample_width_){
```

Comment:
`./apollo.sh lint` fails as missing space at the end of L77-79
`cpplint.py modules/drivers/microphone/microphone_component.cc` gives:

```
modules/drivers/microphone/microphone_component.cc:77:  Missing space before {  [whitespace/braces] [5]
modules/drivers/microphone/microphone_component.cc:78:  Missing space before {  [whitespace/braces] [5]
modules/drivers/microphone/microphone_component.cc:79:  Missing space before {  [whitespace/braces] [5]
Done processing modules/drivers/microphone/microphone_component.cc
Total errors found: 3
```

Reply:
Fixed~